app/models/loan.rb

```diff
@@ -211,4 +212,12 @@ def healthy?
   def health_status_available?
     return !health_check.nil?
   end
+
+  def agents_the_same?
+    (primary_agent.present? || secondary_agent.present?) && primary_agent == secondary_agent
+  end
+
+  def check_agents
+    errors.add(:primary_agent, "can't be the same as secondary agent") if agents_the_same?
```

Comment:
We'll replace this error with an error from the yml file. Let's use this error message for now:

"The point person for this project cannot be the same as the second point person."